live_pipeline/schema.sql

```diff
@@ -0,0 +1,98 @@
+-- cleanup
+
+DROP TABLE IF EXISTS review;
+DROP TABLE IF EXISTS genre;
+DROP TABLE IF EXISTS game;
+DROP TABLE IF EXISTS developer;
+DROP TABLE IF EXISTS publisher;
+DROP TABLE IF EXISTS platform;
+
```

Comment:
General Question: Do we need to create a database for these tables to sit in? E.g.

```
DROP DATABASE IF EXISTS steampulse;
CREATE DATABASE steampulse;
\c steampulse;
```